src/PAModel/EditorState/CombinedTemplateState.cs

```diff
@@ -50,6 +52,18 @@ internal class CombinedTemplateState
         // Maps to TemplateMetadataJson ExtensionData
         public Dictionary<string, object> ComponentExtraMetadata { get; set; }
 
+        public bool IsWidgetTemplate
```

Comment:
Can you add a comment about what this means?

nit:
IsWidgetTemplate {get;set;} 
and skip the field.